packages/server/package.json

```diff
@@ -18,7 +18,9 @@
     "clean": "rimraf dist",
     "dev": "ts-node-dev --poll --respawn --transpile-only --require ./src/otel/instrumentation.ts src/index.ts",
     "start": "node --require ./dist/otel/instrumentation.js dist/index.js",
-    "test": "jest"
+    "test:seed:serial": "jest seed-serial.test.ts --config jest.seed.config.ts --coverageDirectory \"<rootDir>/coverage/seed/serial\"",
+    "test:seed:parallel": "jest seed.test.ts --config jest.seed.config.ts --coverageDirectory \"<rootDir>/coverage/seed/parallel\"",
+    "test": "npm run test:seed:parallel && jest"
```

Comment:
I understand this, but I don't love it.

One of my main testing patterns is to run individual tests using `npm t -- src/file.ts -t 'test name'` - that will be significantly slower, because it will always run `seed.test.ts` first, right?

Given that the test seed is a rare event, I'd rather just add it to the developer setup docs.